src/components/TransactionItemsBox.vue

```diff
@@ -7,18 +7,20 @@
                     From:
                     <v-avatar size="30px">
                         <img
-                                v-if="props.item.from.avatar"
+                                v-if="!props.item.to.avatar.includes('scontent')"
```

Comment:
looks like we should display `from.avatar` instead of `to.avatar` (like it was before) -- or the label above is wrong